components/chart/linearTimeSeries.js

```diff
@@ -0,0 +1,127 @@
+(function(d3, fc) {
+    'use strict';
+
+    fc.charts.linearTimeSeries = function() {
+
+        var xScale = d3.time.scale();
+        var yScale = d3.scale.linear();
+        var xAxis = d3.svg.axis()
+            .scale(xScale)
+            .orient('bottom');
+        var yAxis = d3.svg.axis()
+            .scale(yScale)
+            .orient('left');
```

Comment:
Minor point, but I think most financial charts use a right-hand axis. I guess the reason being that you are more likely to be interested in the most recent prices.